OmniBLE/Bluetooth/PeripheralManager+OmniBLE.swift

```diff
@@ -7,16 +7,10 @@
 //
 
 
-protocol MessageResult {
-    
-}
-
-struct MessageSendFailure: MessageResult {
-    var error: Error
-}
-
-struct MessageSendSuccess: MessageResult {
-    
```

Comment:
Good to see the MessageResult protocol, the MessageSendFailure, and the MessageSendSuccess structs go away. I had prototyped a simpler way to do this using no return type for sendMessage() which throws on errors, but this won't be needed with this approach using a SendMessageResult return.